velox/functions/sparksql/Hash.cpp

```diff
@@ -131,6 +134,8 @@ std::shared_ptr<SparkVectorHasher<HashClass>> createVectorHasher(
       return std::make_shared<MapVectorHasher<HashClass>>(decoded);
     case TypeKind::ROW:
       return std::make_shared<RowVectorHasher<HashClass>>(decoded);
+    case TypeKind::UNKNOWN:
+      return std::make_shared<UnknowTypeVectorHasher<HashClass>>(decoded);
```

Comment:
This is also used by xxhash64. Could you confirm if the behavior is the same? Perhaps also add tests in velox/functions/sparksql/tests/XxHash64Test.cpp.

Reply:
done